packages/perseus/src/widgets/passage-ref-target/passage-ref-target.tsx

```diff
@@ -67,7 +67,6 @@ export default {
     transform: (editorProps: any): any => {
         return _.pick(editorProps, "content");
     },
-    version: {major: 0, minor: 0},
```

Comment:
There's no point in having a 0.0 version I don't think since `version` itself is optional.